### ISSUE TITLE
Blockchains (or Wallet) can’t be on VirtualBox shared folders

### ISSUE THREAD

**Issue by borneq** [dashpay/dash#2129]
I have host system Windows 10 and Linux MIn 18.3 na VirtualBox Guest
I have shared Windows folder and I mount it:
sudo mount -t vboxsf vbshare ~/host
I prefer several blockchains have in shared folders, it is especially imprtant for Bitcoin because has 211 GB od data, and I can share between my Virtual Linuxes. Also I can share Litecoin.
I can’t share Dash and I must all in Linux, I also can’t copy from Linux to Windows and copy to another Virtual LInux due to errors. I must in Linux download from scratch.
Maybe problem is with BerkeleyDB and LevelDB enables it? I read that blocks are with LevelDB but Wallet no. Is possible have wallet on other directory than -datadir (as argument)?

**Comment by jhodges10** [comment]
I don't think it has anything to do with LevelDB / BerkeleyDB. Should always be possible to do a symlink anyways if you just need to trick your system into thinking the folder is in one place when it's actually in another.
